src/components/dashboards/InHetKort.vue

```diff
@@ -114,6 +109,14 @@ export default {
 }
 </script>
 
-<!-- Add "scoped" attribute to limit CSS to this component only -->
 <style lang="scss" scoped>
+  @import "~stijl/src/styles/generic/ams-colorpalette.scss";
+
+  .align-center {
+    text-align: center;
```

Comment:
This seems to be a application style, why not include all global styles in app.scss. The lesser component specific styling the easier the maintenance

Reply:
good one, i cleaned it up a bit